docs/misc/projects.rst

```diff
@@ -239,3 +241,13 @@ Playing Pokemon Red with Reinforcement Learning.
 | Author: Peter Whidden
 | Github: https://github.com/PWhiddy/PokemonRedExperiments
 | Video: https://www.youtube.com/watch?v=DcYLT37ImBY
+
+
+Evolving Reservoirs for Meta Reinforcement Learning
+---------------------------------------------------
+
+A simplified framework to study how neural structures can be optimized at the scale of evolution, in order to enhance the capabilities of agents to learn complex tasks at the scale of development.
```

Comment:
In simpler words, what does that mean/do?

Reply:
Mmh it this better ? (Im' sorry if it is not, this is kind of a weird project to explain)

"Meta-RL framework to optimize reservoir-like neural structures (special kind of RNNs), and integrate them to RL agents to improve their training."

Reply:
that's better, maybe specify which aspect is improved?

Reply:
Ok I added that !